test/torchaudio_unittest/transforms/autograd_test_impl.py

```diff
@@ -137,6 +137,16 @@ def test_amplitude_to_db(self):
         waveform = get_whitenoise(sample_rate=sample_rate, duration=0.05, n_channels=2)
         self.assert_grad(transform, [waveform])
 
+    def test_melscale(self):
+        sample_rate = 8000
+        n_fft = 400
+        n_mels = n_fft // 2 + 1
+        transform = T.MelScale(sample_rate=sample_rate, n_mels=n_mels)
+        waveform = get_spectrogram(
```

Comment:
Can you rename `waveform` to `spectrogram` or `spec`?
This might appear trivial, but this is rather important for readability.